legal/terms-of-service.md

```diff
@@ -52,7 +52,7 @@ as of 05/2018
 
 1. Adjustments, changes and updates of the Travis CI Software that help to avoid or maintain dysfunctions of the Travis CI Software may lead to temporary service suspensions. Travis CI will try to limit downtime of the service or restrictions of accessibility to 10 hours a month. Travis CI will try to do regular maintenance works during the weekend or at times between 10 p.m. and 6 a.m. (CET). Outside of the said hours, Travis CI will inform the Customer about upcoming maintenance works.
 2. The Customer is aware that the service relies on a working internet infrastructure. Additional downtime of the service can occur if the website is not available and at any other time with restrictive access to the internet of the Customer.
-3. The Customer is aware that the Travis CI Software does not work if GitHub is not properly available (be it to Travis CI or the Customer).
+3. The Customer is aware that the Travis CI Software does not work if GitHub/Atlassian Bitbucket is not properly available (be it to Travis CI or the Customer).
```

Comment:
Im wondering, maybe we can replace Github/Atlassian with some generic name for example 'Proveider*.We are going to add Gitlab/assembla support soon so we will have to update it again

Reply:
we have to wait for the legal department's decision on that, but good point :-)